package.json

```diff
@@ -95,6 +95,7 @@
         "restify-cors-middleware2": "2.2.1",
         "restify-errors": "8.0.2",
         "restify-logger": "2.0.1",
+        "restifyapigenerate": "^1.1.0",
```

Comment:
Could you remove the caret to keep dependency versions fixed. To upgrade deps, run `npm run update`